src/core/simulation_store.jl

```diff
@@ -34,6 +34,24 @@ struct SimulationStoreStageParams
     resolution::Dates.Period
     base_power::Float64
     system_uuid::Base.UUID
+
+    function SimulationStoreStageParams(
+        num_executions,
+        horizon,
+        interval,
+        resolution,
+        base_power,
+        system_uuid,
+    )
+        new(
+            num_executions,
+            horizon,
+            Dates.Millisecond(interval),
```

Comment:
In some tests I was getting Dates.Hour here, so I added code to force them to Millisecond.